### ISSUE TITLE
Assumptions for lang and xml:lang confusing

### ISSUE THREAD

**Issue by moekraft** [act-rules/act-rules.github.io#961]
Assumptions
Although there is no known way that an inappropriate secondary subtag in the lang or xml:lang attribute (such as "en-XYZ") can lead to accessibility issues, it is conceivable that with specific assistive technologies on some languages, there can be exceptions. In that case lang and xml:lang should have matching secondary subtags.

I'm a bit confused by this assumption. This rule applies to lang and xml:lang tags that have valid subtags, however, we are saying that in some cases there may be inappropriate secondary subtags and that these should match.lease describe the issue with references and relevant examples where necessary.

--- or ---

If authoring a proposal for a new rule, choose from one of the issue templates:
- [Atomic Rule Template](https://raw.githubusercontent.com/act-rules/act-rules.github.io/develop/pages/design/atomic-template-empty.md)
- [Composite Rule Template](https://raw.githubusercontent.com/act-rules/act-rules.github.io/develop/pages/design/composite-template-empty.md)

**Comment by kasperisager** [comment]
This is being changed in #934: https://github.com/act-rules/act-rules.github.io/pull/934/files#diff-9d3b2c107950b4eba0295294cb9ed080R35-R37

**Comment by jkodu** [comment]
Slightly related issue - https://github.com/act-rules/act-rules.github.io/issues/668 & PR #934  is being worked on.
